pyproject.toml

```diff
@@ -45,7 +45,7 @@ compat = [
 ]
 
 [tool.uv]
-required-version = "0.9.5"
+required-version = ">=0.9.5"
```

Comment:
Seen together, the reason the version is pinned is to ensure that the python versions shipped with uv work with the repo.
Undo the change and add a comment to explain the pin